pkg/cmd/cli/cmd/rollback.go

```diff
@@ -0,0 +1,82 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kubecmd "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd"
+
+	latestapi "github.com/openshift/origin/pkg/api/latest"
+	client "github.com/openshift/origin/pkg/cmd/client"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+)
+
+const rollbackLongDesc = `
+Rolls back a deploymentConfig to match a previous deployment. By default, only the pod
+template of the specified deployment will be included in the rollback.
+
+Examples:
+  $ %[1]s %[2]s deployment-2
+  ...
+
+  $ %[1]s %[2]s deployment-3 --include-strategy=true
+  ...
+
+  $ %[1]s %[2]s deployment-1 --output=json
+  ...
+`
+
+func NewCmdRollback(parentName string, name string, f *Factory, out io.Writer) *cobra.Command {
+	rollback := &deployapi.DeploymentConfigRollback{}
+
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("%s <from-deployment>", name),
+		Short: "Rolls back a deploymentConfig to match a previous deployment.",
+		Long:  fmt.Sprintf(rollbackLongDesc, parentName, name),
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) == 0 || len(args[0]) == 0 {
+				usageError(cmd, "A deployment name is required.")
+			}
+
+			rollback.Spec.From.Name = args[0]
+
+			output := kubecmd.GetFlagString(cmd, "output")
+
+			osClient, _, err := f.Clients(cmd)
+			checkErr(err)
+
+			namespace := getOriginNamespace(cmd)
+
+			newConfig, err := osClient.DeploymentConfigs(namespace).Rollback(rollback)
+			checkErr(err)
+
+			if len(output) == 0 {
+				_, updateErr := osClient.DeploymentConfigs(namespace).Update(newConfig)
+				checkErr(updateErr)
```

Comment:
It only exits on error, and I want to return always in this case. I changed the code structure and added a comment to better convey the intent.